nimare/diagnostics.py

```diff
@@ -139,7 +150,6 @@ def transform(self, result):
         # Use study IDs in inputs_ instead of dataset, because we don't want to try fitting the
         # estimator to a study that might have been filtered out by the estimator's criteria.
         meta_ids = estimator.inputs_["id"]
-        rows = ["Center of Mass"] + list(meta_ids)
 
         # Let's label the clusters in the thresholded map so we can use it as a NiftiLabelsMasker
```

Comment:
Currently `get_clusters_table` only return the clusters table, so we need to re-run `ndimage.label` (reproducing the code in `get_clusters_table`). There is a current PR in nilearn to return the label map, but that won't be available until the next release.

Reply:
could you link this pull request in another issue so we can revisit this when nilearn makes another release?